src/common/utxobased/engine/makeUtxoEngine.ts

```diff
@@ -141,6 +141,8 @@ export async function makeUtxoEngine(
 
   const engine: EdgeCurrencyEngine = {
     async accelerate(edgeTx: EdgeTransaction): Promise<EdgeTransaction | null> {
+      if (currencyInfo.canReplaceByFee !== true) return null
```

Comment:
Although this way is longer, I would rather make the default `canReplaceByFee` value explicit:

```typescript
const { canReplaceByFee = false} = currencyInfo
if (!canReplaceByFee) return null
```